CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ set(SSG_TABLE_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/tables")
 set(SSG_ANSIBLE_ROLE_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/scap-security-guide/ansible")
 set(SSG_BASH_ROLE_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/scap-security-guide/bash")
 set(SSG_KICKSTART_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/scap-security-guide/kickstart")
+set(OSCAP_INSTALL_DATADIR "/usr/share/openscap")
+set(OSCAP_XCCDF_XSL "${OSCAP_INSTALL_DATADIR}/xsl/xccdf_1.1_to_1.2.xsl")
```

Comment:
OSCAP_XCCDF_XSL should be an `option` otherwise it is not customizable.

And there are many XSL in `/usr/share/openscap/xsl`, I suggest making variable name more specific.

Reply:
Actually, `OSCAP_INSTALL_DATADIR`, should be the customizable variable.